serverless/pages/custom-roles.mdx

```diff
@@ -0,0 +1,105 @@
+---
+slug: /serverless/custom-roles
+title: Custom roles
+description: Create and manage roles that grant privileges within your project.
+tags: [ 'serverless', 'Elasticsearch', 'Observability', 'Security' ]
+---
+
+{/* <DocIf condition={"((serverlessCustomRoles))" === "false"}>
+<DocBadgeComingSoon>Coming soon</DocBadgeComingSoon>
+</DocIf> */}
+
+{/* TO-DO: Change feature flag before merging */}
+{/* <DocIf condition={"((serverlessCustomRoles))" === "true"}> */}
+<DocBadge template="technical preview" />
+This content applies to: <DocBadge template="elasticsearch" slug="/serverless/elasticsearch/what-is-elasticsearch-serverless" /> <DocBadge template="observability" slug="/serverless/observability/what-is-observability-serverless" /> <DocBadge template="security" slug="/serverless/security/what-is-security-serverless" />
+
+The built-in <DocLink slug="/serverless/general/assign-user-roles" section="organization-level-roles">organization-level roles</DocLink> and <DocLink slug="/serverless/general/assign-user-roles" section="instance-access-roles">instance access roles</DocLink> are great for getting started with ((serverless-full)), and for system administrators who do not need more restrictive access.
+{/* Derived from https://www.elastic.co/guide/en/kibana/current/tutorial-secure-access-to-kibana.html#_roles */}
+
+{/*
+TBD: Accurate for serverless? Which types of users have access to this UI?
+The `manage_security` cluster privilege is required to manage custom roles.
+*/}
+As an administrator, however, you have the ability to create your own roles to describe exactly the kind of access your users should have.
+For example, you might create a marketing_user role, which you then assign to all users in your marketing department.
+This role would grant access to all of the necessary data and features for this team to be successful, without granting them access they don't require.
+{/* Derived from https://www.elastic.co/guide/en/kibana/current/tutorial-secure-access-to-kibana.html */}
+
+All custom roles grant the same access as the `Viewer` instance access role with regards to ((ecloud)) privileges.
+To grant more ((ecloud)) privileges, assign more roles.
+Users receive a union of all their roles' privileges.
+
+You can manage custom roles in **((project-settings)) → ((manage-app)) →((custom-roles-app))**.
+To clone, delete, or edit a role, open the actions menu:
+
+![Custom Roles app](../images/custom-roles-ui.png)
+{/* TO-DO: This screenshot needs to be refreshed and automated. */}
+
+Roles are a collection of privileges that enable users to access project features and data.
+For example, when you create a custom role, you can assign ((es)) cluster and index privileges and ((kib)) privileges.
+
+<DocCallOut>
+  You cannot assign [run as privileges](((ref))/security-privileges.html#_run_as_privilege) in ((serverless-full)) custom roles.
+</DocCallOut>
```

Comment:
Should we mention that remote cluster and index privileges are also not available?

Reply:
That's being handled in https://github.com/elastic/elasticsearch/pull/109718